vue-frontend/src/components/Authentication.vue

```diff
@@ -36,7 +36,7 @@
         </v-card-media>
         <v-card-actions>
           <v-spacer />
-          <v-btn flat class="orange--text" :rules="[students.username!=''&& students.password!='' ? true : false]" :loading="studentLoading" type="submit">Sign In</v-btn>
+          <v-btn flat class="orange--text" :rules="[students.username!=''&& students.password!='' ? true : false]" :loading="this.studentType ? pending : false" type="submit">Sign In</v-btn>
```

Comment:
Just do `pending`. Also don't use `this` in the template scope, it will always resolve to false I believe.

Reply:
If I don't add `: false`, it complains that `:loading="studentType ? pending"` is invalid expression, but if I do `:loading="studentType ? pending : false"`, it works as intended (doesn't always resolve to false).

Reply:
Ah, you can do `:loading="studentType && pending"`. Also you'll want to add the disabled state to the other button. A user shouldn't be able to submit the student form and then the educator form before the student response is received.